scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -246,6 +246,9 @@ def make_extract_table(control_samples,
         if not (bool(re.match(r"[a-z0-9_-]+$", key)) & bool(re.match(r"[a-z0-9_-]+$", value))):
           raise ValueError(f"label key or value did not pass validation--format should be 'key1=val1, key2=val2'")
 
+    # add labels for DSP Cloud Cost Control Labeling and Reporting
+    query_labels.update({'service':'gvs','team':'variants','managedby':'prepare_ranges_callset'})
+
```

Comment:
dont forget to make a new docker image to use these changes

Reply:
good catch — thanks!